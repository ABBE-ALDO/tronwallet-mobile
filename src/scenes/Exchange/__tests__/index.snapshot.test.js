import React from 'react'
import { shallow } from 'enzyme'
import { ExchangeScene } from './../index'
import exchangeListMock from './../../../services/__mocks__/exchangeList.json'

jest.mock('./../../../utils/i18n')
jest.mock('./../../../utils/sentryUtils')
jest.mock('./../../../store/context')

describe('Exchange List Snapshot', () => {
  let exchangeSceneWrapper = null
  beforeEach(() => {
    const props = { }

    exchangeSceneWrapper = shallow(<ExchangeScene {...props} />)
  })

  test('Should match snapshot', () => {
    exchangeSceneWrapper.setState({ transactions: exchangeListMock })
    expect(exchangeSceneWrapper).toMatchSnapshot()
  })
})
