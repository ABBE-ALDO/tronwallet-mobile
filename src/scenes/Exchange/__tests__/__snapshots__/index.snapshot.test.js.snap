// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exchange List Snapshot Should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ExchangeScene />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "background": "#191A2A",
      "children": Array [
        <NavigationHeader
          isSearching={false}
          onSearch={[Function]}
          onSearchPressed={[Function]}
          searchPreview="Search by the Token name"
          title="Exchange"
        />,
        <LoadingScene />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "isSearching": false,
          "onSearch": [Function],
          "onSearchPressed": [Function],
          "searchPreview": "Search by the Token name",
          "title": "Exchange",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "background": "#191A2A",
        "children": Array [
          <NavigationHeader
            isSearching={false}
            onSearch={[Function]}
            onSearchPressed={[Function]}
            searchPreview="Search by the Token name"
            title="Exchange"
          />,
          <LoadingScene />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "isSearching": false,
            "onSearch": [Function],
            "onSearchPressed": [Function],
            "searchPreview": "Search by the Token name",
            "title": "Exchange",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenThreeAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
